src/app/group/add-edit-group/add-edit-group.component.html

```diff
@@ -1 +1,71 @@
-<p>add-edit-group works!</p>
+<!-- Group Add and Edit Form  -->
+<form [formGroup]="addEditGroup">
+
+    <!-- Form Header  -->
+    <div class="modal-header">
+
+        <!-- If clicked on add button -->
+        <h1 class="modal-title fs-5" id="exampleModalLabel" *ngIf="router.url == '/group/add'">Add Details</h1>
+
+        <!-- If clicked on add button -->
+        <h1 class="modal-title fs-5" id="exampleModalLabel" *ngIf="router.url == '/group/edit'">Edit Details</h1>
+
```

Comment:
Use data binding for h1 heading, create and use function for router url and display text Add/Edit accordingly

Reply:
Use data binding for Add/Edit heading like {{heading}}, and handle it's value in component if isGroupAddActive then heading should be Add Details, if not then Edit Details. you can handle this with one variable no need to create another variable isGroupEditActive. still if you not understand ask me